fdbserver/SimulatedCluster.actor.cpp

```diff
@@ -758,7 +758,8 @@ void SimulationConfig::generateNormalConfig(int minimumReplication, int minimumR
 		}
 	default:
 		ASSERT(false); // Programmer forgot to adjust cases.
-	}
+	} */
+	set_config("ssd-rocksdb-experimental");
```

Comment:
didn't mean to commit this, but this is how I'm forcing simulation to use rocks for now.